content/id/docs/setup/production-environment/tools/kubeadm/install-kubeadm.md

```diff
@@ -0,0 +1,307 @@
+---
+title: Menginstal kubeadm
+content_type: task
+weight: 10
+card:
+  name: setup
+  weight: 20
+  title: Menginstal perkakas setup kubeadm
+---
+
+<!-- overview -->
+
+<img src="https://raw.githubusercontent.com/kubernetes/kubeadm/master/logos/stacked/color/kubeadm-stacked-color.png" align="right" width="150px">Laman ini menunjukkan cara untuk menginstal  perkakas `kubeadm`.
+Untuk informasi mengenai cara membuat sebuah klaster dengan kubeadm setelah kamu melakukan proses instalasi ini, lihat laman [Menggunakan kubeadm untuk Membuat Sebuah Klaster](/docs/setup/production-environment/tools/kubeadm/create-cluster-kubeadm/).
+
+
+
+## {{% heading "prerequisites" %}}
+
+
+* Satu mesin atau lebih yang menjalankan:
+  - Ubuntu 16.04+
+  - Debian 9+
+  - CentOS 7
+  - Red Hat Enterprise Linux (RHEL) 7
+  - Fedora 25+
+  - HypriotOS v1.0.1+
+  - Container Linux (teruji pada versi 1800.6.0)
+* 2 GB RAM atau lebih per mesin (kurang dari nilai tersebut akan menyisakan sedikit ruang untuk
+   aplikasi-aplikasimu)
+* 2 CPU atau lebih
+* Koneksi internet pada seluruh mesin pada klaster (kamu dapat menggunakan internet
+  publik ataupun pribadi)
+* _Hostname_ yang unik, alamat MAC, dan product_uuid untuk setiap Node. Lihat [di sini](#memastikan-alamat-mac) untuk detail lebih lanjut.
+* Port tertentu pada mesin. Lihat [di sini](#check-required-ports) untuk detail lebih lanjut.
```

Comment:
port can be translated to https://kbbi.web.id/porta
also, anchor link has changed
```suggestion
* Porta tertentu pada mesin. Lihat [di sini](#memeriksa-porta-yang-dibutuhkan) untuk detail lebih lanjut.
```

Reply:
Ah, porta is the formal version of port. Fixed in 0b74a869bea33a4eb86e43a89b5284bcd244444f